bindings/node.js/test/kzg.test.ts

```diff
@@ -81,6 +81,35 @@ function bytesEqual(a: Uint8Array | Buffer, b: Uint8Array | Buffer): boolean {
   return true;
 }
 
+function getValidTest(testDir: string): any {
+  const tests = globSync(testDir);
+  const validTest = tests.find(
+    (testFile: string) =>
+      !testFile.includes("invalid") && !testFile.includes("incorrect") && !testFile.includes("different")
+  );
+  if (!validTest) throw new Error("Could not find valid test");
+  return yaml.load(readFileSync(validTest, "ascii"));
+}
+
+function testArgCount(fn: (...args: any[]) => any, validArgs: any[]): void {
+  const lessArgs = validArgs.length === 1 ? [] : validArgs.slice(0, -1);
```

Comment:
```suggestion
  const lessArgs = validArgs.slice(0, -1);
```
e.g. `[1].slice(0,-1)` is already `[]`

Reply:
Good catch. Updated.